driver/src/price_finding/naive_solver.rs

```diff
@@ -47,9 +51,8 @@ impl Order {
         exec_sell_amount: u128,
     ) -> U256 {
         // TODO - Refer to Alex's Lemma [ceil(p/float(q)) == (p + q - 1) // q]
```

Comment:
I think this line can go. The fact that the integer division (p + q - 1) // q equals the Ceil it probably well known. However we should document this some how. Perhaps just as a note here.

```
// ceil(p/float(q)) == (p + q - 1) // q
```

Reply:
```suggestion
        // Note that: ceil(p / float(q)) == (p + q - 1) // q
```